front/components/assistant_builder/AssistantBuilderDataSourceModal.tsx

```diff
@@ -138,19 +139,63 @@ function PickDataSource({
   show: boolean;
   onPick: (dataSource: DataSourceType) => void;
 }) {
+  const [query, setQuery] = useState("");
+
+  // Order in the following format : connectorProvider > empty > webcrawler
+  const orderedDataSources = dataSources.sort((a, b) => {
+    const aConnector = a.connectorProvider;
+    const bConnector = b.connectorProvider;
+
+    const order = Object.keys(CONNECTOR_CONFIGURATIONS)
+      .filter(
+        (key) =>
+          CONNECTOR_CONFIGURATIONS[key as keyof typeof CONNECTOR_CONFIGURATIONS]
+            .connectorProvider !==
+          CONNECTOR_CONFIGURATIONS.webcrawler.connectorProvider
+      )
+      .map(
+        (key) =>
+          CONNECTOR_CONFIGURATIONS[key as keyof typeof CONNECTOR_CONFIGURATIONS]
+            .connectorProvider
+      );
+
+    if (aConnector === CONNECTOR_CONFIGURATIONS.webcrawler.connectorProvider) {
```

Comment:
Doesn't it place webcrawler websites at the top?

Reply:
Nop, when webcrawler is `a` then the `b` element should be upper (so I return 1)

Reply:
Ok!